File: java script/milky doughnut/TaskFive.js
function removeDuplicates(array) {
    let temp = [];
    let uniqueCount = 0;

    for (let current = 0; current < array.length; current++) {
        let found = false;
        for (let checked = 0; checked < uniqueCount; checked++) {
            if (temp[checked] === array[current]) {
                found = true;
                break;
            }
        }
        if (!found) {
            temp[uniqueCount] = array[current];
            uniqueCount++;
        }
    }
    return temp;
}

File: python/milky doughnut/task_five.py

def remove_duplicates(array):
    temp = []
    for currentNumber in range(len(array)):
        found = False
        for checked in range(len(temp)):
            if temp[checked] == array[currentNumber]:
                found = True
                break
        if not found:
            temp.append(array[currentNumber])
    return temp








